pulse_lib/virtual_channel_constructors.py

```diff
@@ -20,28 +30,59 @@ def __init__(self, pulse_lib_obj, name=None):
         self.virtual_gate_names = []
         self._virtual_gate_matrix = None
         self.valid_indices = None
+        self.square = square
+        if inverse:
+            if square:
+                raise ValueError('Cannot have inverse and square true.')
```

Comment:
Is this correct? defaults are square=True and inverse=True. So, with defaults this raises an exception.